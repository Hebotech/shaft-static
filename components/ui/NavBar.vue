<template>
  <vs-navbar text-white :color="active" center-collapsed v-model="active">
    <template #left>
      <img src="/shaft-logo.svg" alt="" />
    </template>
    <template #right>
      <vs-navbar-item :active="active == 'primary'" id="primary">
        Primary
      </vs-navbar-item>
      <vs-navbar-item :active="active == 'success'" id="success">
        Success
      </vs-navbar-item>
      <vs-navbar-item :active="active == 'danger'" id="danger">
        Danger
      </vs-navbar-item>
      <vs-navbar-item :active="active == 'warn'" id="warn">
        Warn
      </vs-navbar-item>
      <vs-navbar-item :active="active == 'dark'" id="dark">
        Dark
      </vs-navbar-item>
      <vs-navbar-item :active="active == '#7d33ff'" id="#7d33ff">
        HEX
      </vs-navbar-item>
      <vs-navbar-item
        :active="active == 'rgb(59,222,200)'"
        id="rgb(59,222,200)"
      >
        RGB
      </vs-navbar-item>
    </template>
  </vs-navbar>
</template>
<script>
export default {
  data: () => ({
    active: 'primary',
  }),
};
</script>
