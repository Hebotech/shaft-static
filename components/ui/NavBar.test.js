import {shallowMount} from '@vue/test-utils';
import NavBar from './navBar.vue';

describe('Navbar ui components', ()=>{
  let cmp = shallowMount(NavBar);
  it('Navbar is the same with the logo', ()=>{
    expect(cmp.element).toMatchSnapshot();
  })
})