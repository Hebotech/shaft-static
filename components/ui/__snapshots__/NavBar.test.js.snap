// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar ui components Navbar is the same with the logo 1`] = `
<nav
  class="navbar navbar-light bg-light bg-transparent"
>
  <a
    class="navbar-brand"
    href="#"
  >
    <img
      alt="splash-Shaft image"
      class="d-inline-block align-top img-fluid"
      src="@/assets/brand/logo.png"
      width="150px"
    />
  </a>
</nav>
`;
